src/test/java/it/gov/pagopa/cgn/portal/controller/profile/GetProfileApiTest.java

```diff
@@ -42,14 +41,15 @@
     void init() {
         agreement = agreementService.createAgreementIfNotExists(TestUtils.FAKE_ID);
         profilePath = TestUtils.getProfilePath(agreement.getId());
+        setOperatorAuth();
     }
 
     @Test
     void Get_GetProfileWithInvalidAgreementId_NotFound() throws Exception {
```

Comment:
We should update the test name
```suggestion
    void Get_GetProfileWithInvalidAgreementId_Forbidden() throws Exception {
```

Reply:
Fixed in the last commit